lld/test/wasm/call-indirect-external.s

```diff
@@ -0,0 +1,27 @@
+# RUN: llvm-mc -filetype=obj -triple=wasm32-unknown-unknown -o %t.o %s
+
+# RUN: not wasm-ld --experimental-pic -shared %t.o -o /dev/null 2>&1 | \
+# RUN:   FileCheck -check-prefix=ERRUND %s
+# ERRUND: error: {{.*}}.o: relocation R_WASM_TABLE_INDEX_REL_SLEB cannot be used against an undefined symbol `external_func`
```

Comment:
Maybe just using `CHECK:` here and then remove all the `-check-prefix=ERRUND` aguments?  (since its the only prefix used in this file).

Reply:
Thank you for the suggestion. I adjusted this.